src/native/libs/System.Native/pal_networking.c

```diff
@@ -2005,9 +2005,11 @@ static bool TryGetPlatformSocketOption(int32_t socketOptionLevel, int32_t socket
                     *optName = TCP_KEEPINTVL;
                     return true;
 
+#ifdef TCP_FASTOPEN_CONNECT
```

Comment:
Why is this included?

Reply:
Yes, it was last week's change https://github.com/am11/runtime/commit/e377f169c0d790172baf762f93dc011f727777e5, the other usage was guarding with this macro, this was an oversight (CI doesn't have anything which was testing these #else cases).